src/input_xml.F90

```diff
@@ -1049,8 +1049,9 @@ subroutine read_geometry_xml()
     do i = 1, n_cells
       c => cells(i)
 
-      ! Initialize the number of cell instances - this is a base case for distribcells
+      ! Initialize distribcell instances and distribcell index
```

Comment:
Shouldn't the `instances` and `distribcell_ind` closely mirror each other? I guess I'm not yet sure what `instances` is, perhaps it is a global counter of all cell instances rather than just those that are catalogued for distribcell filters?